libnavigation-ui/build.gradle

```diff
@@ -3,7 +3,6 @@ apply plugin: 'kotlin-android'
 apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.jaredsburrows.license'
-apply plugin: 'com.mapbox.android.sdk.versions'
```

Comment:
Is it safe to remove https://github.com/mapbox/android-sdk-versions-plugin 🤔? cc @harvsu

Reply:
Good catch! The semantic versioning has been blocking me when using the plugin because it requires the global variables to be set which impacted the scripts. I can try re-adding the plugin and extending the scripts to make it work again.

Reply:
Removing this plugin would affect the reformed user Agent in Events okhttp request. This generator plugin generates a file that feeds the Nav SDK version to the reformed user agent. c.c @alfwatt